### DISCUSSION TITLE
From what language did English borrow the word "typhoon"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#249] [Galaxy Trivia] [answered]
**Hint:** 🇭🇰

**Comment by thuongtruong1009** [comment, marked as answer]
The word "typhoon" in English was borrowed from the Chinese language. Specifically, it comes from the Chinese word "tai fung," which means "great wind." The word was first introduced into the English language through the Portuguese, who had contact with Chinese sailors and merchants in the South China Sea and brought the word back to Europe in the 16th century.
